xml/tuning-os-tuning.xml

```diff
@@ -86,244 +91,254 @@
-   <sect2>
-     <title>SSD Tuning</title>
-     <para>
-       To get the best read performance, it may be necessary to adjust
-       the <option>read_ahead</option> and write cache settings for the SSD devices.
-       In our particular testing environment, disabling write cache
-       and forcing <option>read_ahead</option> to 2&nbsp;MB resulted in the best
-       overall performance.
-     </para>
-     <para>
-       Before tuning, it is important to check the default values and
-       measure the differences in performance compared to the baseline.
-     </para>
-     <para>
-       By placing the following file in <filename>/etc/udev/rules.d</filename>
-       the device is detected by the model name shown in
-       <filename>/sys/block/{devname}/device/model</filename> and assigned to
-       disable write caching and setting the <option>read_ahead_kb</option> to 2&nbsp;MB.
-     </para>
+   <para>
+    The &salt; state and steps used in this testing can be found in
+    <xref linkend="tuning-appendix-a"/>. This needs to be adjusted to work in
+    each customer environment. An example of adjusting the
+    <literal>grub2</literal> configuration file can also be found in the
+    appendix.
+   </para>
+  </sect2>
+
+  <sect2>
+   <title>I/O tuning - Multiqueue Block I/O</title>
+   <para>
+    The first aspect to tune is to ensure that I/O if flowing in the most
+    optimal pattern. For the test cluster used in this test, that means enable
+    multi-queue block I/O. This is done through adding a boot-time kernel
+    parameter as found in the Section 12.4
+    <link xlink:href="https://documentation.suse.com/sles/15-SP1/html/SLES-all/cha-tuning-io.html#cha-tuning-io-barrier"/>.
+    This is not an action that should be taken unilaterally on clusters that
+    contain spinning media devices due to potential performance degradation for
+    those devices. The general result is that there are multiple I/O queues
+    assigned to the device, allowing more jobs to be handled simultaneously by
+    those such as NVMe and SSD that can service large numbers of requests.
+   </para>
+  </sect2>
+
+  <sect2>
+   <title>SSD Tuning</title>
+   <para>
+    To get the best read performance, it may be necessary to adjust the
+    <option>read_ahead</option> and write cache settings for the SSD devices.
+    In our particular testing environment, disabling write cache and forcing
+    <option>read_ahead</option> to 2&nbsp;MB resulted in the best overall
+    performance.
+   </para>
+   <para>
+    Before tuning, it is important to check the default values and measure the
+    differences in performance compared to the baseline.
+   </para>
+   <para>
+    By placing the following file in <filename>/etc/udev/rules.d</filename> the
+    device is detected by the model name shown in
+    <filename>/sys/block/{devname}/device/model</filename> and assigned to
+    disable write caching and setting the <option>read_ahead_kb</option> to
+    2&nbsp;MB.
```

Comment:
```suggestion
    By placing the following file in <filename>/etc/udev/rules.d</filename>,
    devices will be detected according to the model name shown in
    <filename>/sys/block/{devname}/device/model</filename>, and instructed to
    disable write caching and set the <option>read_ahead_kb</option> option to
    2&nbsp;MB.
```

Reply:
I note that no filename seems to be specified here. I think we should at least offer an example filename.